roles/inspec/defaults/main.yml

```diff
@@ -5,15 +5,27 @@ inspec:
   interval: 3600
   dependency_install_method: 'tar'
   dependency_path: /etc/inspec/
-  profile_dependencies:
-    - name: inspec-openstack-security
+  profile_installations:
```

Comment:
I'm not sure `profile_installations` is a good description?   `profile_location`  or `profile_metadata` would seem to make more sense to me.